### PR TITLE
[ready] Update London Heathrow

### PR DESCRIPTION
**Pull request by eliuuk**
London Heathrow is the UK's most major airport, and the busiest out of the 6 London airports. It came 6th busiest in terms of passenger movements in 2015.

Checklist as outlined by @Fechulo:
- [x] Update STAR's
- [x] Terrain
- [x] Improve traffic

### CODE REVIEW

**Comment by jakobeng1303** [comment]
Are you coding the transitions for 27R/L as well?

**Comment by indianbhaji** [comment]
@jakobeng1303 There are no transitions?

**Comment by erikquinn** [comment]
@indianbhaji Could you verify everything is working right with this? On my end, I'm not getting any arrivals to spawn when I merge this. Not quite sure what the story is with that, because I get no errors in the console, but definitely no arrivals either.

**Comment by eliuuk** [comment]
@erikquinn Oops! The _ on arrivals needs to be removed :p

**Comment by eliuuk** [comment]
@erikquinn Fixed!

**Comment by erikquinn** [comment]
@indianbhaji Did you mean to add terrain here? It's checked above, but there is no terrain present.

**Comment by Fechulo** [comment]
@erikquinn there is no terrain near Heathrow that's higher than 1000ft so that's probably why no terrain was added

**Comment by erikquinn** [comment]
@Fechulo well that would make sense! lol thanks

**Comment by erikquinn** [comment]
Conflicts resolved; merged via c709861fdcc3c1969dd03506acc19a7ec454c91e.

File: assets/airports/egll.json [hunk 1]
```diff
@@ -997,117 +1014,456 @@
...
     "destinations": [
       "CPT", "BPK", "WOBUN", "DET", "GOGSI", "MAY", "MID"
     ],
     "type": "random",
     "offset": 0,
-    "frequency": 40
+    "frequency": 44
+  },
+  "stars": {
+    "_comment": "In case the BIG  VOR is out of service",
```
Comment (erikquinn) [inline assets/airports/egll.json:1082]: @indianbhaji For future reference, please keep any comments like this within the star entry, as in `"BIG": {"_comment": "words words words", "icao": "BIG", ...}` due to some recent changes. :smiley:

**Pull request closed without merging**